golang/examples/Prove.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"errors"
+
+	btcspv "github.com/summa-tx/bitcoin-spv/golang/btcspv"
+)
+
+func Prove(version []byte, vin []byte, vout []byte, locktime []byte, merkleRoot []byte, intermediateNodes []byte, index uint) (string, error) {
```

Comment:
this doesn't need to return an error, for some reason, I thought that `btcsp.Prove`  bubbled up an error